xclim/indicators/atmos/_temperature.py

```diff
@@ -786,3 +787,28 @@ class TasminTasmax(Daily2D):
     missing="skip",
     compute=indices.corn_heat_units,
 )
+
+biologically_effective_degree_days = TasminTasmax(
+    identifier="biologically_effective_degree_days",
+    units="K days",
+    long_name="Biologically effective degree days computed with {method} formula (Summation of min((max((Tmin + Tmax)/2"
+    " - {thresh_tasmin}, 0) * k) + TR_adg, 9°C), for days between {start_date} and {end_date}).",
+    description="Heat-summation index for agroclimatic suitability estimation, developed specifically for viticulture. "
+    "Considers daily Tmin and Tmax with a base of {thresh_tasmin} between 1 April and 31 October, with a maximum daily "
+    "value for degree days (typically 9°C). It also integrates a modification coefficient for latitudes "
+    "between 40°N and 50°N as well as swings in daily temperature range.",
+    cell_methods="",
+    comment="Original formula published in Gladstones, 1992.",
+    compute=wrapped_partial(
+        indices.biologically_effective_degree_days,
+        suggested=dict(
+            thresh_tasmin="10 degC",
+            method="gladstones",
+            low_dtr="10 degC",
+            high_dtr="13 degC",
+            max_daily_degree_days="9 degC",
+            start_date="04-01",
+            end_date="11-01",
+        ),
+    ),
+)
```

Comment:
Thanks for the explanation. I wasn't sure if this should be its own indicator class, given that lat is a DataArray/coordinate.

Reply:
Currently your question makes sense, but with #736 it will all be caduc because that PR will lower the need for intermediate indicator classes a lot. You are right that you could have made a `TasminTasmaxLat` class and added datachecks and cfchecks for `lat` (as it is considered a variable input), but that will be done automatically with the other PR. And, we don't have any checks for lat currently anyway.